masterlist.yaml

```
@@ -8820,6 +8820,31 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507/'
+        name: 'Woodworker''s Whim'
+    group: *craftingGroup
+    after: [ 'Complete Crafting Overhaul_Remastered.esp' ]
+    msg:
+      - <<: *patchProvided
+        subs:
+          - 'Campfire - Complete Camping System'
```

Single element maps should be written in flow style:

```suggestion
        subs: [ 'Campfire - Complete Camping System' ]
```